ingress/controllers/gce/main.go

```diff
@@ -266,6 +267,15 @@ func getClusterUID(kubeClient *client.Client, name string) (string, error) {
 	}
 	for _, ing := range ings.Items {
 		if len(ing.Status.LoadBalancer.Ingress) != 0 {
+			fwRuleName, ok := ing.Annotations[fmt.Sprintf("%v/forwarding-rule", utils.K8sAnnotationPrefix)]
+			if ok {
+				uidList := strings.Split(fwRuleName, clusterNameDelimiter)
```

Comment:
Kind of wary of this parsing logic here - the code/utility that created the fwRuleName string in the annotation should provide a parse function to convert it back into its constituent components ?